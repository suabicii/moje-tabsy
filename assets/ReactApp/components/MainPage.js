import React from "react";

function MainPage() {
    return (
        <div>
            <p className="lead">Hello from MainPage</p>
        </div>
    );
}

export default MainPage;