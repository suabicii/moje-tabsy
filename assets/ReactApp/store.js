import {configureStore} from "@reduxjs/toolkit";
import drugsSlice from "./features/drugs/drugsSlice"
import userSlice from "./features/user/userSlice";
import darkModeSlice from "./features/darkMode/darkModeSlice";

export default configureStore({
    reducer: {
        drugs: drugsSlice,
        user: userSlice,
        darkMode: darkModeSlice
    }
});