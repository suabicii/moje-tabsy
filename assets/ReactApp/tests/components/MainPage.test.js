import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import MainPage from "../../components/MainPage";
import {Provider} from "react-redux";
import store from "../../store";

it('should correctly render MainPage', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(
        <Provider store={store}>
            <MainPage/>
        </Provider>
    );

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});