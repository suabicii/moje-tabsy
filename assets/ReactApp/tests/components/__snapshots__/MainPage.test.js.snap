// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render MainPage 1`] = `
<main
  className="dashboard__main"
>
  <div
    className="container py-4"
  >
    <AppRouter />
  </div>
</main>
`;
